src/mscorlib/shared/System/IO/PathInternal.Windows.cs

```diff
@@ -343,6 +405,13 @@ internal static string NormalizeDirectorySeparators(string path)
         {
             if (string.IsNullOrEmpty(path)) return path;
 
+            return new string(NormalizeDirectorySeparators(path.AsReadOnlySpan()));
+        }
+
+        internal static ReadOnlySpan<char> NormalizeDirectorySeparators(ReadOnlySpan<char> path)
+        {
+            if (path.IsEmpty) return path;
```

Comment:
I think our style requires the return on the next line (not necessraily with { } )